ui/app/components/transaction-view/transaction-view.component.js

```diff
@@ -65,12 +65,57 @@ export default class TransactionView extends Component {
     }
   }
 
+  toggleAccountWarning = () => {
+    localStorage.set('accountWarningDismissed', true)
+  }
+
+  renderAccountConsolidation = () => {
+    const { identities } = this.props
+    const copy = identities
```

Comment:
Should this line be: 
`const copy = { ...identities }`
for a safe clone?

Reply:
pushed a commit with said changes.